spec/datadog/appsec/configuration/settings_spec.rb

```diff
@@ -52,7 +52,7 @@ def patcher
         context 'is not defined' do
           let(:appsec_enabled) { nil }
 
-          it { is_expected.to be described_class::DEFAULT_APPSEC_ENABLED }
+          it { is_expected.to eq false }
         end
```

Comment:
:clap: I really love having the defaults inlined in the tests. That's because it protects us from changing some default accidentally, or at least without considering its impact on the tests -- if the behavior of the implementation (default) changes, the tests should have to change too!